spec/controllers/nonprofits/nonprofit_keys_spec.rb

```diff
@@ -11,6 +11,14 @@
 
     describe 'mailchimp_login' do
       include_context :open_to_np_associate, :get, :mailchimp_login, nonprofit_id: :__our_np
+
+      it 'properly redirects to a mailchimp domain' do
+        sign_in user_as_np_admin
+
+        get :mailchimp_login, params: {nonprofit_id: nonprofit.id}
+
+        expect(response).to redirect_to(%r{https://login.mailchimp.com.*})
```

Comment:
Aren't regexes fun! Those dots in the domain will match any character unless they're escaped:

```suggestion
        expect(response).to redirect_to(%r{https://login\.mailchimp\.com.*})
```

Reply:
(Regexes are actually awesome, but of course there are some pitfalls like this :smile:)

Reply:
*slaps forehead*

Of course they will! I switched to `%r{` to work around the `/`'s but forgot about the periods.